inst/HTM0/HTM0_whyUseMizer.Rmd

```diff
@@ -77,108 +79,112 @@ We can do the same with Biomass to see when, if any, of the species collapse. Fo
+plot_Brel <- ggplot(Brel_select) +
+    geom_line(aes(x = time, y = value)) +
+    theme_pubr() +
+    geom_hline(yintercept = 0.1, colour = "red", size = 0.75) +
+    geom_hline( yintercept = 0.5, colour = "steel blue", size = 0.75 ) +
+    labs(x = "", y = "Relative Biomass") +
+    scale_y_continuous(limits = c(0, max(Brel_select$value)))
 
 plot_Brel<- plot_Brel  +
-annotate("text",x = 1951, y = 0.1+0.05, label = "Blim",color="red")+
-annotate("text",x = 1951, y = 0.5+0.05, label = "Bmsy",color="steel blue")+
-annotate("text",x = 2015, y = max(Brel_select$value), label = select_species,color="black") 
+    annotate("text", x = 1951, y = 0.1+0.05, label = "Blim", color = "red") +
+    annotate("text", x = 1951, y = 0.5+0.05, label = "Bmsy", color = "steel blue") +
+    annotate("text", x = 2015, y = max(Brel_select$value), 
+             label = select_species, color = "black") 
 
 plot_Brel
 
 # which years < 0.1 B_unfished
 
 Brel_select <- Brel %>%
-  filter(sp == "Sprat" & time >= 1950) %>%
-  mutate(collapsed = value < 0.1)
+    filter(sp == "Sprat" & time >= 1950) %>%
+    mutate(collapsed = value < 0.1)
 
 ggplot(Brel_select, aes(x = time, y = value, fill = collapsed)) +
-  geom_col(position = "identity") +
-  geom_hline(yintercept = 0.1, linetype=1, colour="dark grey", size=0.75) +
-  scale_fill_manual(values = alpha(c("blue", "red"), .3)) +
-  theme_pubr()
+    geom_col(position = "identity") +
+    geom_hline(yintercept = 0.1, colour = "dark grey", size = 0.75) +
+    scale_fill_manual(values = alpha(c("blue", "red"), .3)) +
+    theme_pubr()
 
 ```
 
 
 ### Projecting future recovery scenarios 
 
-We may now wish to explore the potential recovery the larger species and sizes in the system. To do this we set  up another scenario, where the model starts with the last time step of the fished scenario.
+We may now wish to explore the potential recovery of the larger species and sizes in the system. To do this we set  up another scenario, where the model starts with the last time step of the fished scenario.
 
 First let's take a look at what the model has used in the historical period (from ICES stock assessments).
 
 ```{r}
-effort<-melt(sim@effort)  %>% filter(time >= 1980)
+effort <- melt(getEffort(sim))  %>% filter(time >= 1980)
 
-plot_effort<-ggplot(effort) + 
-geom_line(aes(x = time, y = value, colour = gear))+ 
-theme_pubr() +
-labs(y="Effort",x="") 
+plot_effort <- ggplot(effort) + 
+    geom_line(aes(x = time, y = value, colour = gear)) + 
+    theme_pubr() +
+    labs(y = "Effort", x = "") 
 
 plot_effort
 ```
 
 The species' maximum fishing mortality rates (which we call for convenience effort) for Cod and Saithe have declined but are not as low as what would be in line with single-species "Fmsy" (a reference level of fishing  deemed sustainable from  single-species stock assessements) and the fishing mortality rate for Sprat and Dab is very high. 
 
-We can the values directly like this:
+The effort values in 2018 were
```

Comment:
Why do you choose 2018 and not 2019 which was the last time step of your simulation?

Reply:
I imagine this is a typo, I'll fix this in my edits.

Reply:
Because there is an NA in dab effort  in 2019 and  haven't fixed it